src/omotes_sdk/internal/worker/params_dict.py

```diff
@@ -0,0 +1,87 @@
+import logging
+from typing import Type, Optional, TypeVar, cast
+
+from omotes_sdk.types import ParamsDict, ParamsDictValues
+
+logger = logging.getLogger("omotes_sdk_internal")
+
+Expected = TypeVar("Expected", bound=ParamsDictValues)
```

Comment:
Maybe change 'Expected' to 'ParamValue'?

Reply:
Good find! Fixed.